### ISSUE TITLE
Unable to capture the screenshot into reportportal using appiumdriver

### ISSUE THREAD

**Issue by krishnaPedd** [reportportal/reportportal#1324]
Hi @HardNorth,
 
Using this below method we are not able to capture screenshots into report portal through appium driver
below is the screenshot method i am using for mobile and it is working fine for Web framework

// Take screenshots into report portal
public static ReportPortalMessage ScreenShot(AppiumDriver appiumDriver) {

			 ReportPortalMessage message = null;
			    try {
			    	TakesScreenshot ts = (TakesScreenshot) appiumDriver;
			    	File srcFile = ts.getScreenshotAs(OutputType.FILE);
			    	java.util.Date d= new java.util.Date();
			    	 org.apache.commons.io.FileUtils.copyFile(srcFile, new File("./ScreenShots/"+d.toString().replace(":", "_")+".png"));
			    	String rp_message = "";//test message for ReportPortal
			        message = new ReportPortalMessage(srcFile, rp_message);
			    } catch (IOException e) {
			        e.printStackTrace();
			        LOGGER.error(e);
			    }
			    LOGGER.info(message);
			    return message;
		}

**Comment by HardNorth** [comment]
@krishnaPedd I doubt WebDriver and AppiumDriver provide screenshots in the same format, but you treat them as PNG in any case.

**Comment by krishnaPedd** [comment]
Yes we are generating screenshots in PNG format only for web and mobile. So the same above approach it was worked for report portal  for web platform, but incase of mobile it is generating screenshot locally but it  is not reflecting in report portal.

**Comment by HardNorth** [comment]
@krishnaPedd As I said most likely it's not PNG.

**Comment by krishnaPedd** [comment]
@HardNorth ,Even we tried with pdf aswell but it didn't worked, please let me know in which format we need to generate screenshots to reflect screenshots in report portal dashboard for mobile scripts.

**Comment by HardNorth** [comment]
@krishnaPedd can you just save a screenshot file and test it somewhere like this: https://www.checkfiletype.com/
?

**Comment by krishnaPedd** [comment]
@HardNorth, i uploaded and verified the screenshot in the above link, it is showing that it's pdf  and png based on the file, the format it is showing as expected only. here my question is, screenshot capturing feature is available for mobile devices in report portal using appium driver?, if yes, which format of screenshot will report portal support?

**Comment by HardNorth** [comment]
@krishnaPedd No, RP doesn't have such feature. And we do not plan to add it. We are reporting tool, not a test framework.

**Comment by MaryiaRymarchyk** [comment]
Hi @krishnaPedd, is the issue still relevant for you?
Seems that there are no remain actions from ReportPortal side.
